test/MatrixSets/MatrixZonotope.jl

```diff
@@ -1,4 +1,5 @@
 using LazySets, Test
+import LazySets.MatrixZonotopeModule: factors
```

Comment:
```suggestion
```
This should not be needed because `factors` is exported.

Also, it should be `using`: `import` of functions is used to extend them. I know it is confusing because `import` of packages has a different meaning.